tests/acceptance/pageObjects/FilesPageElement/sharingDialog.js

```diff
@@ -531,6 +532,14 @@ module.exports = {
           disabled = result.value
         })
       return disabled
+    },
+    /**
+     * checks if the required label is present in the expiration date field
+     * @return Boolean
+     */
+    isRequiredLabelPresent: async function () {
```

Comment:
not needed, in the context `waitForElementVisible('@requiredLabelInCollaboratorsExpirationDate')` can be used

Reply:
fixed